pkg/kube_events_manager/resource_informer.go

```diff
@@ -296,6 +296,7 @@ func (ei *resourceInformer) HandleWatchEvent(object interface{}, eventType Watch
 		object = staleObj.Obj
 	}
 	obj := object.(*unstructured.Unstructured)
+	oldObj := object.(*unstructured.Unstructured)
```

Comment:
```suggestion
	oldObj := oldObject.(*unstructured.Unstructured)
```

Reply:
But the check is required that the oldObj is not nil.

Reply:
I have updated my code as your suggestions.!